Form/Type/ImageType.php

```diff
@@ -56,15 +56,15 @@ public function configureOptions(OptionsResolver $resolver)
     }
 
     /**
-     * {@inheritdoc}
+     * @return string
```

Comment:
this also leads to a symfony warning if we don't do it?

Reply:
yep https://github.com/franmomu/LiipImagineBundle/runs/4102778452?check_suite_focus=true#step:11:132